### DISCUSSION TITLE
AI人工智慧在現實世界中之應用

### DISCUSSION THREAD

**Discussion by 107602505** [1101-GS4719-C/Python-Community-GS4719-C#149] [科技新知]
人工智慧的應用已逐漸滲透到各行各業，尤其在市場行銷上特別有感，因為在搜尋引擎上輸入商品後，就會在各個社群網站(Facebook、instagram)跳出類似的商品廣告，利用客戶的點閱紀錄客製化推播相關產品，省時又省力且能精準預測客戶的愛好及消費習慣。

新聞網址:
[https://udn.com/news/story/6871/5430229](https://udn.com/news/story/6871/5430229)

**Comment by HSU-YU-JEN** [comment]
對~平常只要在瀏覽過某些網購或是訂票等網站，後續的網站就會是這些商品的廣告內容

**Comment by chupochun** [comment]
現在到處都能看見人工智慧的影子，真的是無所不在

**Comment by pekoranoinu** [comment]
AI已經成為各產業不可或缺的部分。

**Comment by ThomasMeng0515** [comment]
AI確實已經進入我們的生活，甚至有些工作可能會被AI的應用所取代。
